src/pretix/api/serializers/item.py

```diff
@@ -16,11 +16,34 @@ class Meta:
                   'position', 'default_price', 'price')
 
 
+class ItemVariationSerializer(I18nAwareModelSerializer):
+    class Meta:
+        model = ItemVariation
+        fields = ('id', 'value', 'active', 'description',
+                  'position', 'default_price', 'price')
+
+
 class InlineItemAddOnSerializer(serializers.ModelSerializer):
     class Meta:
         model = ItemAddOn
         fields = ('addon_category', 'min_count', 'max_count',
-                  'position')
+                  'position', 'price_included')
```

Comment:
Would you mind adding this new field to the documentation and examples in ``items.rst`` as well (together with a ``versionchanged`` block)? :)